### Eclipse Workspace Patch 1.0
#P jquery-ui-trunk
Index: ui/ui.draggable.js
===================================================================
--- ui/ui.draggable.js	(revision 1535)
+++ ui/ui.draggable.js	(working copy)
@@ -396,10 +396,17 @@
 
 		var inst = $(this).data("draggable");
 		inst.sortables = [];
-		$(ui.options.connectToSortable).each(function() {
-			// 'this' points to a string, and should therefore resolved as query, but instead, if the string is assigned to a variable, it loops through the strings properties,
-			// so we have to append '' to make it anonymous again
-			$(this+'').each(function() {
+		
+		var strSortable;
+		if (ui.options.connectToSortable instanceof Array)
+			strSortable = ui.options.connectToSortable.join(',');
+		else
+			strSortable = ui.options.connectToSortable;
+		
+		$(strSortable).each(function() {
+//			// 'this' points to a string, and should therefore resolved as query, but instead, if the string is assigned to a variable, it loops through the strings properties,
+//			// so we have to append '' to make it anonymous again
+			$(this).each(function() {
 				if($.data(this, 'sortable')) {
 					var sortable = $.data(this, 'sortable');
 					inst.sortables.push({
