### Eclipse Workspace Patch 1.0
#P jquery-ui-trunk
Index: ui/ui.draggable.js
===================================================================
--- ui/ui.draggable.js	(revision 1734)
+++ ui/ui.draggable.js	(working copy)
@@ -433,6 +433,11 @@
 
 		var inst = $(this).data("draggable");
 		inst.sortables = [];
+		
+		if (typeof ui.options.connectToSortable === 'string') {
+			ui.options.connectToSortable = [ui.options.connectToSortable];
+		}
+		
 		$(ui.options.connectToSortable).each(function() {
 			// 'this' points to a string, and should therefore resolved as query, but instead, if the string is assigned to a variable, it loops through the strings properties,
 			// so we have to append '' to make it anonymous again
