Index: ui.datepicker-hr.js
===================================================================
--- ui.datepicker-hr.js	(revision 2310)
+++ ui.datepicker-hr.js	(working copy)
@@ -6,14 +6,16 @@
 		prevText: '&#x3c;',
 		nextText: '&#x3e;',
 		currentText: 'Danas',
-		monthNames: ['Siječanj','Veljača','Ožujak','Travanj','Svibanj','Lipani',
+		monthNames: ['Siječanj','Veljača','Ožujak','Travanj','Svibanj','Lipanj',
 		'Srpanj','Kolovoz','Rujan','Listopad','Studeni','Prosinac'],
 		monthNamesShort: ['Sij','Velj','Ožu','Tra','Svi','Lip',
 		'Srp','Kol','Ruj','Lis','Stu','Pro'],
-		dayNames: ['Nedjalja','Ponedjeljak','Utorak','Srijeda','Četvrtak','Petak','Subota'],
+		dayNames: ['Nedjelja','Ponedjeljak','Utorak','Srijeda','Četvrtak','Petak','Subota'],
 		dayNamesShort: ['Ned','Pon','Uto','Sri','Čet','Pet','Sub'],
 		dayNamesMin: ['Ne','Po','Ut','Sr','Če','Pe','Su'],
-		dateFormat: 'dd.mm.yy.', firstDay: 1,
-		isRTL: false};
+		dateFormat: 'dd.mm.yy.',
+		firstDay: 1,
+		isRTL: false
+	};
 	$.datepicker.setDefaults($.datepicker.regional['hr']);
-});
\ No newline at end of file
+});
