Index: tests/unit/button/button_events.js
===================================================================
--- tests/unit/button/button_events.js	(revision 3788)
+++ tests/unit/button/button_events.js	(working copy)
@@ -14,4 +14,22 @@
 	ok( set.find("label:first").click().is(".ui-button") );
 });
 
+test("click (set with bind)", function() {
+	expect(2);
+	$("#button").button().bind("click", function() {
+		ok( true );
+	});
+	ok( $("#button").click().is(".ui-button"), "Click event should have been fired" );
+});
+
+test("click (set as option)", function() {
+	expect(2);
+	$("#button").button({
+		click: function() {
+			ok( true );
+		}
+	});
+	ok( $("#button").click().is(".ui-button"),  "Click event should have been fired" );
+});
+
 })(jQuery);
