Index: ui.datepicker-zh-CN.js
===================================================================
--- ui.datepicker-zh-CN.js	(revision 486)
+++ ui.datepicker-zh-CN.js	(working copy)
@@ -5,9 +5,9 @@
 		clearText: '清除', clearStatus: '清除已选日期',
 		closeText: '关闭', closeStatus: '不改变当前选择',
 		prevText: '&#x3c;上月', prevStatus: '显示上月',
-		prevBigText: '&#x3c;&#x3c;', prevBigStatus: '',
+		prevBigText: '&#x3c;&#x3c;', prevBigStatus: '显示上一年',
 		nextText: '下月&#x3e;', nextStatus: '显示下月',
-		nextBigText: '&#x3e;&#x3e;', nextBigStatus: '',
+		nextBigText: '&#x3e;&#x3e;', nextBigStatus: '显示下一年',
 		currentText: '今天', currentStatus: '显示本月',
 		monthNames: ['一月','二月','三月','四月','五月','六月',
 		'七月','八月','九月','十月','十一月','十二月'],
Index: ui.datepicker-zh-TW.js
===================================================================
--- ui.datepicker-zh-TW.js	(revision 486)
+++ ui.datepicker-zh-TW.js	(working copy)
@@ -5,9 +5,9 @@
 		clearText: '清除', clearStatus: '清除已選日期',
 		closeText: '關閉', closeStatus: '不改變目前的選擇',
 		prevText: '&#x3c;上月', prevStatus: '顯示上月',
-		prevBigText: '&#x3c;&#x3c;', prevBigStatus: '',
+		prevBigText: '&#x3c;&#x3c;', prevBigStatus: '顯示上一年',
 		nextText: '下月&#x3e;', nextStatus: '顯示下月',
-		nextBigText: '&#x3e;&#x3e;', nextBigStatus: '',
+		nextBigText: '&#x3e;&#x3e;', nextBigStatus: '顯示下一年',
 		currentText: '今天', currentStatus: '顯示本月',
 		monthNames: ['一月','二月','三月','四月','五月','六月',
 		'七月','八月','九月','十月','十一月','十二月'],
