Index: ui.progressbar.js
===================================================================
--- ui.progressbar.js	(revision 498)
+++ ui.progressbar.js	(working copy)
@@ -27,14 +27,14 @@
 			rangeValue: 0,
 			threads: {},
 			
-			wrapper: $("<div class='ui-progressbar-wrap'></div>"),
-			bar: $("<div class='ui-progressbar-bar ui-hidden'></div>").css({
+			wrapper: $('<div class="ui-progressbar-wrap"></div>'),
+			bar: $('<div class="ui-progressbar-bar ui-hidden"></div>').css({
 				width: '0px', overflow: 'hidden', zIndex: 100
 			}),
-			textElement: $("<div class='ui-progressbar-text'></div>").html(text).css({
+			textElement: $('<div class="ui-progressbar-text"></div>').html(text).css({
 				width: '0px', overflow: 'hidden'
 			}),
-			textBg: $("<div class='ui-progressbar-text ui-progressbar-text-back'></div>").html(text).css({
+			textBg: $('<div class="ui-progressbar-text ui-progressbar-text-back"></div>').html(text).css({
 					width: this.element.css('width')
 			})
 			
