Index: ui/ui.datepicker.js
===================================================================
--- ui/ui.datepicker.js	(revision 872)
+++ ui/ui.datepicker.js	(working copy)
@@ -246,6 +246,7 @@
 		$.data(target, PROP_NAME, inst);
 		this._setDate(inst, this._getDefaultDate(inst));
 		this._updateDatepicker(inst);
+		this._updateAlternate(inst);
 	},
 	
 	/* Tidy up after displaying the date picker. */
