*** jquery-ui-1.7.1/jquery-ui-1.7.1/ui/jquery-ui.js	2009-03-18 23:18:58.000000000 -0500
--- jquery-ui-1.7.1/ui/jquery-ui.js	2009-03-20 11:52:46.000000000 -0500
***************
*** 7662,7667 ****
--- 7662,7671 ----
  		if (this.instances.length === 0) {
  			$([document, window]).unbind('.dialog-overlay');
  		}
+         else
+         {
+             $(this.instances[this.instances-1]).unbind('.dialog-overlay');
+         }
  
  		$el.remove();
  	},
