Index: ui.dialog.js
===================================================================
--- ui.dialog.js	(revision 2130)
+++ ui.dialog.js	(working copy)
@@ -163,11 +163,15 @@
 		(self.overlay && self.overlay.destroy());
 		self.uiDialog.unbind('keypress.ui-dialog');
 
-		(self.options.hide
-			? self.uiDialog.hide(self.options.hide, function() {
+		if (self.options.hide) {
+			self.uiDialog.hide(self.options.hide, function() {
 				self._trigger('close', event);
-			})
-			: self._trigger('close', event));
+			});
+		}
+		else {
+			self.uiDialog.hide();
+			self._trigger('close', event);
+		}
 
 		$.ui.dialog.overlay.resize();
 
