Index: effects.core.js
===================================================================
--- effects.core.js	(revision 2450)
+++ effects.core.js	(working copy)
@@ -175,7 +175,7 @@
 	},
 
 	toggle: function(){
-		if(!arguments[0] || (arguments[0].constructor == Number || (/(slow|normal|fast)/).test(arguments[0])) || (arguments[0].constructor == Function))
+		if(!arguments[0] || arguments[0] === true || (arguments[0].constructor == Number || (/(slow|normal|fast)/).test(arguments[0])) || (arguments[0].constructor == Function))
 			return this.__toggle.apply(this, arguments);
 		else {
 			return this.effect.apply(this, _normalizeArguments(arguments, 'toggle'));
